src/test/java/org/broadinstitute/consent/http/resources/DataAccessRequestResourceTest.java

```diff
@@ -250,10 +241,7 @@ void testUpdateByReferenceIdForbidden() {
     try {
       when(userService.findUserByEmail(any())).thenReturn(invalidUser);
       when(dataAccessRequestService.findByReferenceId(any())).thenReturn(dar);
-      resource =
-          new DataAccessRequestResource(daaService,
-              dataAccessRequestService, emailService, gcsService, userService, datasetService,
-              matchService);
+      initResource();
```

Comment:
It might be possible to move all of these `initResource();` calls into a `@BeforeEach` method and simplify this more.

Reply:
Good call!